tests/integration/targets/vmware_rest_appliance/tasks/appliance_localaccounts.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: Get the initial global policy of the local accounts
+  vmware.vmware_rest.appliance_localaccounts_globalpolicy_info:
+  register: initial_global_policy
+
+- name: Assert initial global policy is retrieved
+  ansible.builtin.assert:
+    that:
+      - initial_global_policy is defined
+      - initial_global_policy.value is not none
+    fail_msg: "Failed to retrieve the initial global policy for local accounts."
+
+- name: Print initial global policy
+  ansible.builtin.debug:
+    var: initial_global_policy
+
+- name: Temporarily update the global policy of the local accounts
+  vmware.vmware_rest.appliance_localaccounts_globalpolicy:
+    warn_days: 3
+
+- name: Validate temporary global policy update
+  vmware.vmware_rest.appliance_localaccounts_globalpolicy_info:
+  register: temp_global_policy
+
+- name: Assert temporary global policy is updated warn_days to 3
+  ansible.builtin.assert:
+    that:
+      - temp_global_policy.value.warn_days == 3
+    fail_msg: "Temporary global policy was not updated to warn_days: 3."
+
+- name: Update the global policy of the local accounts to 5
+  vmware.vmware_rest.appliance_localaccounts_globalpolicy:
+    warn_days: 5
+
+- name: Validate global policy update warn_days to 5
+  vmware.vmware_rest.appliance_localaccounts_globalpolicy_info:
+  register: updated_global_policy
+
+- name: Assert global policy is updated warn_days to 5
+  ansible.builtin.assert:
+    that:
+      - updated_global_policy.value.warn_days == 5
+    fail_msg: "Global policy was not updated to warn_days: 5."
+
+- name: Reapply the global policy of the local accounts (idempotency check)
+  vmware.vmware_rest.appliance_localaccounts_globalpolicy:
+    warn_days: 5
+  register: idempotency_result
+
+- name: Ensure idempotency works
+  ansible.builtin.assert:
+    that:
+      - not(idempotency_result.changed)
+    fail_msg: "Idempotency check failed. The global policy update triggered changes unnecessarily."
+
+- name: List and validate local accounts
+  vmware.vmware_rest.appliance_localaccounts_info:
+  register: local_accounts
+
+- name: Assert local accounts are listed
+  ansible.builtin.assert:
+    that:
+      - local_accounts is defined
+      - local_accounts.value is not none
+    fail_msg: "Failed to retrieve the list of local accounts."
+
+- name: Print local accounts
+  ansible.builtin.debug:
+    var: local_accounts
```

Comment:
You didn't restore warn_days. Add always block please to restore warn_days to original value.

Reply:
fixed